winui/DataGrid/FilterRow.md

```diff
@@ -190,6 +210,19 @@ The default filter condition is Begins With, the below filter conditions are ava
 <td>
 Always equals filter condition will be applied for filtering the CheckBox value.
 </td>
+<td>
+The default filter condition is Equals, the below filter conditions are available in date filter.
```

Comment:
The default filter condition is Equals, and the following filter conditions are available in the date filter.

Reply:
Hi @Chrispineagunja ,

I have modified the content as per the concern provided.